oak-store-composite/src/main/java/org/apache/jackrabbit/oak/composite/MountInfoConfig.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.jackrabbit.oak.composite;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import org.apache.felix.scr.annotations.Activate;
+import org.apache.felix.scr.annotations.Component;
+import org.apache.felix.scr.annotations.Property;
+import org.apache.felix.scr.annotations.PropertyUnbounded;
+import org.apache.felix.scr.annotations.Service;
+import org.apache.jackrabbit.oak.commons.PropertiesUtil;
+import org.osgi.framework.BundleContext;
+
+import static java.util.stream.Collectors.toList;
+
+@Component(label = "Apache Jackrabbit Oak MountInfo Config", configurationFactory = true, metatype = true)
```

Comment:
For new components rather use the official OSGi component annotations: https://github.com/apache/jackrabbit-oak/blob/9a2cf021772ac577d161f1409ada6edd9642a441/oak-parent/pom.xml#L548

Reply:
Compare also with the ongoing effort tracked in https://issues.apache.org/jira/browse/OAK-6741. Mixing both approaches in the same module shouldn't be a problem.

Reply:
Thanks for your suggestion, @kwin. I have migrated the classes touched by this PR to the official OSGi component annotations.